study-builder/studies/circadia/study.conf

```diff
@@ -0,0 +1,672 @@
+)
+) && (
+user.studies["circadia"].forms["PREQUAL"].questions["CHILD_AND_PREGNANCY"].answers.hasOption("CAP_NONE_OF_THE_ABOVE")
+) && (
+user.studies["circadia"].forms["PREQUAL"].questions["PROBABLE_MEDICAL_PROBLEMS_2"].answers.hasOption("PMP2_NONE_OF_THE_ABOVE")
+&& (
+!user.studies["circadia"].forms["PREQUAL"].questions["PROBABLE_MEDICAL_PROBLEMS_2"].answers.hasOption("PMP2_GINGIVITIS")
+&& !user.studies["circadia"].forms["PREQUAL"].questions["PROBABLE_MEDICAL_PROBLEMS_2"].answers.hasOption("PMP2_XEROSTOMIA")
+&& !user.studies["circadia"].forms["PREQUAL"].questions["PROBABLE_MEDICAL_PROBLEMS_2"].answers.hasOption("PMP2_PERIODONTITIS")
+)
+) && (
+user.studies["circadia"].forms["PREQUAL"].questions["TRAVEL"].answers.hasOption("TRAVEL_NO")
+) && (
+user.studies["circadia"].forms["PREQUAL"].questions["8_TH_GRADE_ENGLISH"].answers.hasOption("8_TH_GRADE_ENGLISH_YES")
+) && (
+(
+user.studies["circadia"].forms["PREQUAL"].questions["VISUAL_PROBLEM_1"].answers.hasOption("VISUAL_PROBLEM_1_YES")
+&& user.studies["circadia"].forms["PREQUAL"].questions["VISUAL_PROBLEM_2"].answers.hasOption("COLOR_BLIND")
+) || (
+user.studies["circadia"].forms["PREQUAL"].questions["VISUAL_PROBLEM_1"].answers.hasOption("VISUAL_PROBLEM_1_NO")
+)
+)
+)
+      """,
+      "maxOccurrencesPerUser": 1,
+      "dispatchToHousekeeping": false,
+      "order": 1
+    },
+    {
+      "trigger": {
+        "type": "ACTIVITY_STATUS",
+        "activityCode": "CONSENT",
+        "statusType": "COMPLETE"
+      },
+      "action": {
+        "type": "ACTIVITY_INSTANCE_CREATION",
+        "activityCode": "HEALTH_AND_LIFESTYLE_QUESTIONNAIRE"
+      },
+      "preconditionExpr": "true",
+      "maxOccurrencesPerUser": 1,
+      "dispatchToHousekeeping": false,
+      "order": 2
+    },
+    {
+      "trigger": {
+        "type": "ACTIVITY_STATUS",
+        "activityCode": "HEALTH_AND_LIFESTYLE_QUESTIONNAIRE",
+        "statusType": "COMPLETE"
+      },
+      "action": {
+        "type": "ACTIVITY_INSTANCE_CREATION",
+        "activityCode": "DLMO_SCHEDULING_INSTRUCTIONS"
+      },
+      "preconditionExpr": "true",
+      "maxOccurrencesPerUser": 1,
+      "dispatchToHousekeeping": false,
+      "order": 3
+    },
+    {
+      "trigger": {
+        "type": "DSM_NOTIFICATION",
+        "dsmEvent": "CIRCADIA_SENT"
+      },
+      "action": {
+        "type": "ACTIVITY_INSTANCE_CREATION",
+        "activityCode": "DLMO_INSTRUCTIONS",
+      },
+      "cancelExpr": "false"
+      "maxOccurrencesPerUser": 1,
+      "dispatchToHousekeeping": false,
+      "order": 1
+    },
+    {
+      "trigger": {
+        "type": "ACTIVITY_STATUS",
+        "activityCode": "HEALTH_AND_LIFESTYLE_QUESTIONNAIRE",
+        "statusType": "COMPLETE"
+      },
+      "action": {
+        "type": "ACTIVITY_INSTANCE_CREATION",
+        "activityCode": "MINI_SLEEP_LOG"
+      },
+      "preconditionExpr": "true",
```

Comment:
`preconditionExpr` and `cancelExpr` are optional. If we don't have a need for them, I recommend just removing it.